src/databases/ADIOS2/avtGTCFileFormat.C

```diff
@@ -59,38 +60,41 @@
 
 using namespace std;
 
-bool
-avtGTCFileFormat::Identify(const char *fname)
+bool avtGTCFileFormat::Identify(const std::string &fname,
```

Comment:
'using namespace std;', yet still specifying std::string, etc.  I have no preference other than consistency. I suggest either remove the using statement or the use of 'std::'. (Here and in other source files).